src/engine/engine_vfs.c

```diff
@@ -179,10 +179,10 @@ int mj_deleteFileVFS(mjVFS* vfs, const char* filename) {
       mju_free(vfs->filedata[i]);
 
       // scroll remaining files forward
-      while (i<vfs->nfile-1) {
-        mjSTRNCPY(vfs->filename[i], vfs->filename[i+1]);
-        vfs->filesize[i] = vfs->filesize[i+1];
-        vfs->filedata[i] = vfs->filedata[i+1];
+      for (int j = i; j < vfs->nfile - 1; ++j) {
```

Comment:
C files in `engine/` have a somewhat [idiosyncratic coding convention](https://github.com/deepmind/mujoco/blob/main/STYLEGUIDE.md). I'm not going to say that I love it, but it is what it is and consistency trumps everything when style is concerned. To wit, `for (int j=i; j<vfs->nfile-1; j++)` would be consistent with the rest of the C code.

This comment does not apply to other code (e.g. test files).

Reply:
Fixed.